cmd/oras/internal/option/remote_test.go

```diff
@@ -43,9 +45,213 @@ var testTagList = struct {
 	Tags: []string{"tag"},
 }
 
+// localhostServerCert is a PEM-encoded TLS cert with SAN IPs
+// "127.0.0.1" and "[::1]", expiring at Jan 29 16:00:00 2084 GMT.
+// adapted from src/crypto/tls:
+// go run generate_cert.go  --rsa-bits 4096 --host 127.0.0.1,::1,oras.land --ca --start-date "Jan 1 00:00:00 1970" --duration=1000000h
+var localhostServerCert = []byte(`-----BEGIN CERTIFICATE-----
```

Comment:
nit: Will it be better to store those testing key pairs as files in the `testdata` folder, and then reference using [`//go:embed`](https://pkg.go.dev/embed)?

Reply:
Great suggestion! learned something new in the process